internal/configs/provider_meta.go

```diff
@@ -31,7 +31,9 @@ func decodeProviderMetaBlock(block *hcl.Block) (*ProviderMeta, hcl.Diagnostics)
 	}
 
 	// verify that the local name is already localized or produce an error.
```

Comment:
This comment should be updated to reflect that it's no longer throwing an error but instead returning early.

Reply:
That's fair. PTAL 🙏🏻